### PR TITLE
[CloudMinds] 10328-SCA PM

### PR DESCRIPTION
**Pull request by Kenny-SZ**
This LWM2M Object includes all PM items belongs to a specific SCA.

### CODE REVIEW

File: 10328.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			<Description><![CDATA[The current of the SCA.]]></Description>
+        </Item>
+	    <Item ID="606">
+			<Name>SCA Temperate</Name>
+			<Operations>R</Operations>
+			<MultipleInstances>Single</MultipleInstances>
+			<Mandatory>Mandatory</Mandatory>
+			<Type>Float</Type>
+			<RangeEnumeration/>
+			<Units>°C</Units>
```
Comment (seanmcilroy29) [inline 10328.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Comment (Kenny-SZ) [reply 10328.xml]: °C has been commented on another object.

Comment (seanmcilroy29) [reply 10328.xml]: Replace ``°C`` with ``Cel``

Comment (Kenny-SZ) [reply 10328.xml]: Changed, thanks.

**Pull request merged**